src/utils/common.ts

```diff
@@ -0,0 +1,65 @@
+import { AxiosError, AxiosInstance, AxiosRequestConfig, default as Axios } from 'axios';
+import { default as axiosRetry, IAxiosRetryConfig } from 'axios-retry';
+import deepmerge from 'deepmerge';
+
+export interface IFacadeConfig {
+    axiosConfig?: AxiosRequestConfig;
+    axiosRetryConfig?: IAxiosRetryConfig;
+    apiKey?: string;
+    maxRetries?: number;
+}
+
+/**
+ * @param {IFacadeConfig} config
```

Comment:
leave only `@throws`

Reply:
why leave only that? better then to remove this block